src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -390,6 +390,15 @@ public Task<Compilation> GetCompilationAsync(CancellationToken cancellationToken
         }
 
         /// <summary>
+        /// Determines if the compilation returned by <see cref="GetCompilationAsync"/> has all the references it's expected to have.
+        /// </summary>
+        // TODO: make this public
+        internal Task<bool> GetCompilationHasIncompleteReferencesAsync(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
For some reason today that sounds better than I think when I last tried it. It'll be HasCompleteReferencesAsync() since it's an async method. I guess the thing that bugs me most here is I want async properties or something...